lab-andrew/test/brewer-router-test.js

```diff
@@ -0,0 +1,153 @@
+'use strict';
+
+const expect = require('chai').expect;
+const request = require('superagent');
+
+const server = require('../server');
+const storage = require('../lib/storage');
+const Brewer = require('../model/brewer');
+
+const port = process.env.PORT || 3000;
+const baseUrl = `localhost:${port}/api/brewer`;
+const createTempBrewer = (done) => {
+  this.tempBrewer = new Brewer('kenya');
+  storage.setItem('brewer', this.tempBrewer);
+  done();
+};
+
+describe('Testing the module brew-router', function() {
+  before((done) => {
+    if (!server.isRunning) {
+      server.listen(port, () => {
+        server.isRunning = true;
+        console.log(`Server running on PORT:${port}`);
+        done();
+      });
+      return;
+    }
+    done();
+  });
+
+  after((done) => {
+    if(server.isRunning) {
+      server.close(() => {
+        console.log('shutdown the server');
+        done();
+      });
+      return;
+    }
+    done();
+  });
+  describe('Testing for bad route', function() {
```

Comment:
This is testing for a bad route, but it's giving you a 404 because the GET request on your endpoint says it doesn't exist, rather than checking to make sure your server is defaulting to the '*' route you specify for everything else which should return the error.